ios/RNMarigoldBridge.m

```diff
@@ -12,18 +12,18 @@ -(instancetype)init {
 
 - (instancetype)initWithJSCodeLocation:(NSURL *)jsCodeLocation
```

Comment:
I think we should simplify these initialisers to only one that just accepts the `jsCodeLocation`. `startEngine` should be handled through the native SDK (as early as possible), same for the GeoIPTracking default. This class should only be concerned with the code location and the list of modules.